tests/integration/test_logcollector/test_configuration/test_basic_configuration_log_format.py

```diff
@@ -264,20 +264,14 @@ def check_log_format_invalid(cfg):
     if cfg['valid_value']:
         pytest.skip('Valid values provided')
 
-    if 'log_format1' in cfg and 'log_format2' in cfg:
-        log_callback = logcollector.callback_multiple_macos_block_configuration()
-        wazuh_log_monitor.start(timeout=5, callback=log_callback,
-                                error_message=gc.GENERIC_CALLBACK_ERROR_MESSAGE)
-    else:
```

Comment:
Why was it removed?

Reply:
This block of code was removed because if you can see, it's on `check_log_format_invalid` function and by the changes about the product, now this is a valid configuration, error message lookup is deprecated because it will not be raised.